src/cclib/io/wfxwriter.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2017, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""A writer for wfx format files."""
+
+import os.path
+
+from . import filewriter
+
+class WFXWriter(filewriter.Writer):
+    """A writer for wfx files."""
+
+    required_attrs = ['atomcoords', 'atomnos', 'gbasis', 'charge', 'homos']
+    keyword = 'GTO'
+
+    @staticmethod
+    def _section(section_name, section_data):
+        """Add opening/closing section_name tags to data."""
+        opening_tag = ['<' + section_name + '>']
+        closing_tag = ['</' + section_name + '>']
+        
+        if isinstance(section_data, list):
+            return opening_tag + section_data + closing_tag
+        elif isinstance(section_data, str):
+            return opening_tag + section_data.split() + closing_tag
+        elif isinstance(section_data, int) or isinstance(section_data, float):
+            return opening_tag + [str(section_data)] + closing_tag
+        else:
+            return opening_tag + closing_tag
```

Comment:
Maybe you can simplify this by making sure all the utility functions return strings instead of ints, floats, and lists?